### PR TITLE
Fix an old function name "rows()" in the vignette about dplyr 0.8.0

### PR DESCRIPTION
**Pull request by yutannihilation**

### CODE REVIEW

File: vignettes/future/dplyr_0.8.0.Rmd [hunk 1]
```diff
@@ -160,7 +160,7 @@ group_data(group_by(df, f))
 group_data(rowwise(df))
 ```
 
-Similarly, the indices themselves can be retrieved using the `rows()` generic: 
+Similarly, the indices themselves can be retrieved using the `group_rows()`
```
Comment (krlmlr) [inline vignettes/future/dplyr_0.8.0.Rmd]: Did you mean to end the line here?

Comment (yutannihilation) [reply vignettes/future/dplyr_0.8.0.Rmd]: Yes. Sorry, I forgot `:`...

**Comment by romainfrancois** [comment]
I’m off in July. Will have a look next month unless someone else merge it first.

**Comment by yutannihilation** [comment]
No hurry at all :)

**Pull request merged**